src/pages/home/report/ReportActionItem.js

```diff
@@ -239,14 +239,17 @@ class ReportActionItem extends Component {
         if (this.props.action.actionName === CONST.REPORT.ACTIONS.TYPE.CHRONOSOOOLIST) {
             return <ChronosOOOListActions action={this.props.action} reportID={this.props.report.reportID} />;
         }
+
+        const hasErrors = !_.isEmpty(this.props.action.errors);
+
         return (
             <PressableWithSecondaryInteraction
                 pointerEvents={this.props.action.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE ? 'none' : 'auto'}
                 ref={el => this.popoverAnchor = el}
                 onPressIn={() => this.props.isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
                 onPressOut={() => ControlSelection.unblock()}
                 onSecondaryInteraction={this.showPopover}
-                preventDefaultContentMenu={!this.props.draftMessage}
+                preventDefaultContentMenu={!this.props.draftMessage && !hasErrors}
```

Comment:
@s77rt Do you think we need to rename this prop name? Should be `preventDefaultContextMenu`.

Reply:
Yes please fix that, it's a typo